app/models/ext_management_system.rb

```diff
@@ -173,6 +192,42 @@ def hostname_format_valid?
 
   default_value_for :enabled, true
 
+  after_save :change_maintenance_for_child_managers, :if => proc { |ems| ems.changed_attributes.include?('enabled') }
+
+  def disable!
+    _log.info("Disabling EMS [#{name}] id [#{id}].")
+    update!(:enabled => false)
+    _log.info("Disabling EMS [#{name}] id [#{id}] successful.")
+  end
+
+  def enable!
+    _log.info("Enabling EMS [#{name}] id [#{id}].")
+    update!(:enabled => true)
+    _log.info("Enabling EMS [#{name}] id [#{id}] successful.")
+  end
+
+  # Move ems to maintenance zone and backup current one
+  def pause!
```

Comment:
What is the effective different between enable/disable and pause/resume?  I understand that there is a literal difference in that pause implies zone move in addition to changing the enabled flag, but outside in when would someone call `pause!` instead of `disable!`?

Reply:
That's good question. `enable!/disable!` has been there before so I don't delete them. They change flag `enabled` only, which causes starting/stopping provider related workers (like refresh worker) without moving provider to another zone. 
`pause!`/`resume!` is terminology according to UI buttons Pause/Resume. 
So I think the question is opposite, do we need `enable!`/`disable` methods?

Reply:
enable/disable methods are now used by API and that PR can block this one for some time, so - as you wrote below, visibility of all of these methods should be changed in next PR.